### ISSUE TITLE
[mesheryctl] `mesheryctl perf result my-test --view` crashes when trying to see performance test results

### ISSUE THREAD

**Issue by mchurichi** [meshery/meshery#5318]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
I'm trying to see the details of the performance test results I ran earlier. I run `mesheryctl perf result my-test --view`, then pick any of the results by typing let's say `0` to choose the first one, and the app crashes showing the error trace below.

#### Desired Behavior
I want to be able to see the results of the test I chose.

#### Screenshots/Logs
```
$ mesheryctl perf result my-test --view
Name: istio_1648736178355
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x2bbc450]

goroutine 1 [running]:
github.com/layer5io/meshery/mesheryctl/internal/cli/root/perf.glob..func5(0x45abde0, 0xc0000a1d20, 0x1, 0x2, 0x0, 0x0)
        /Users/runner/work/meshery/meshery/mesheryctl/internal/cli/root/perf/result.go:137 +0x4b0
github.com/spf13/cobra.(*Command).execute(0x45abde0, 0xc0000a1ce0, 0x2, 0x2, 0x45abde0, 0xc0000a1ce0)
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:856 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0x45a7f60, 0x34bc020, 0x4793c30, 0x0)
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:974 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:902
github.com/layer5io/meshery/mesheryctl/internal/cli/root.Execute()
        /Users/runner/work/meshery/meshery/mesheryctl/internal/cli/root/root.go:68 +0x56
main.main()
        /Users/runner/work/meshery/meshery/mesheryctl/cmd/mesheryctl/main.go:22 +0x25
```

#### Environment

- Host OS: Mac
- Meshery Server Version: stable-v0.6.0-rc-4
- Meshery Client Version: stable-v0.6.0-rc-5h
- Platform: Kubernetes

---

#### Contributor Resources
- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by alphaX86** [comment]
TL;DR - This occurs when provider is set to 'None' instead of 'Meshery' in login screen. Got the error now, fixing it....
